Tools/CISupport/ews-app/ews/common/github.py

```diff
@@ -199,7 +199,8 @@ class GitHubEWS(GitHub):
     ICON_BUILD_FAIL = u'\U0000274C'
     ICON_BUILD_WAITING = u'\U000023F3'
     ICON_BUILD_ONGOING = u'![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png)'
-    ICON_BUILD_ONGOING_WITH_FAILURES = ICON_BUILD_ONGOING  # FIXME: Update icon for this case
+    ICON_BUILD_ONGOING_WITH_FAILURES = u'![loading-orange](https://github-production-user-asset-6210df.s3.amazonaws.com/3098702/291015173-08c448be-ac0a-4fd6-92a3-8165057445b7.png)'
```

Comment:
I think our plan-of-record for making this more accessible should really just be https://github.com/WebKit/WebKit/pull/17187. I think web-based code review tools are intrinsically less accessible than text based ones, and I don't see good solutions other than reverting to a fully text-based review workflow.